src/modules/popUp.js

```diff
@@ -0,0 +1,32 @@
+import displayComments from './displayComments.js';
+import addComment from './addComment.js';
+
+const addEventToCommentBtn = () => {
+  const getAllBtn = document.querySelectorAll('.commentBtn');
+  getAllBtn.forEach((item) => {
+    // eslint-disable-next-line func-names
+    item.addEventListener('click', function () {
+      document.querySelector('.comments .qty').innerHTML = 0;
+      document.querySelector('.popup ul').innerHTML = '';
+      document.querySelector('body').classList.add('active');
+      const getItemId = this.getAttribute('data-idBtn');
+      document.querySelector('.popup').setAttribute('data-popid', getItemId);
+      // console.log(getItemId, 'popup iD');
```

Comment:
It is a best practice to avoid having console.log() between your codes on GitHub. Also have unused code deleted instead of commenting them out to make your code clean